applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -161,7 +174,8 @@ def GetDefaultParameters(cls):
             },
             "problem_domain_sub_model_part_list": [""],
             "processes_sub_model_part_list": [""],
-            "auxiliary_variables_list" : []
+            "auxiliary_variables_list" : [],
+            "use_orthogonal_subscales" : false
```

Comment:
Same comment. The unique elements that support this are the explicit ones. I think it should be placed in the explicit defaults.

Reply:
Ok.